### ISSUE TITLE
Windowmove --relative $wid 0 0 moves window / window border not accounted for

### ISSUE THREAD

**Issue by mxmlnkn** [jordansissel/xdotool#115]
When trying to slide the window horizontally by using relative coordinates e.g. in 1px steps I noticed, that the window would move to the bottom right by way more than 1 px. I tried

    xdotool windowmove --relative $wid 1 0

After that I tried

    xdotool windowmove --relative $wid 1 0

and noticed that the window would move. Here is to show by how much it moved:

    xdotool getwindowgeometry --shell $wid
    xdotool windowmove --relative $wid 0 0
    xdotool getwindowgeometry --shell $wid

The output is:

    X=684
    Y=216
    WIDTH=857
    HEIGHT=445
    SCREEN=0

    X=700
    Y=276
    WIDTH=857
    HEIGHT=445
    SCREEN=0

I very strongly suspect that it is related to the skin (X-aero slate) I am using. Unfortunately this seems to be a general problem with skins. Here is the output directly after changing to the Albatross window theme (xfce)

    X=695
    Y=268
    WIDTH=857
    HEIGHT=445
    SCREEN=0

    X=701
    Y=312
    WIDTH=857
    HEIGHT=445
    SCREEN=0

Note that X decreased by 5 and Y bx 2 directly after changing the window skins, without moving the window!! Also the WIndow now moves `+(6,44)px` instead of `+(16,60)px` with the X Aero Slate theme.

**Comment by mxmlnkn** [comment]
See e.g. the complexity to just move a window back to it's original location:

    getBorderWidth() {
        # argument is window id
        local X Y X0 Y0
        eval $(xdotool getwindowgeometry --shell $1 | command grep '[XY]=')
        X0=$X
        Y0=$Y
        xdotool windowmove --relative $1 0 0
        eval $(xdotool getwindowgeometry --shell $1)
        BORDER_WIDTH=$((X-X0))
        BORDER_HEIGHT=$((Y-Y0))
        xdotool windowmove $1 $((X0-BORDER_WIDTH)) $((Y0-BORDER_HEIGHT))
    }

**Comment by jordansissel** [comment]
Yeah i agree this behavior is not really intuitive. It would be better if
xdotool window move were more aware of window managers so that it would
move ... "More correctly" (one pixel left should mean exactly that, hehe)

**Comment by donjan** [comment]
Duplicate of #95
